webpack.config.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ["style-loader", "css-loader"],
+      },
+    ],
+  },
+	plugins: [
+    new HtmlWebpackPlugin({
+			template: './src/index.html',
+    }),
+  ],
+};
```

Comment:
[REQUIRED] Kindly set up your webpack for development and production. You can check this [link](https://webpack.js.org/guides/production/) out